django/templatetags/i18n.py

```diff
@@ -468,6 +481,13 @@ def do_block_translate(parser, token):
                 six.reraise(TemplateSyntaxError, TemplateSyntaxError(msg), sys.exc_info()[2])
         elif option == "trimmed":
             value = True
+        elif option == "asvar":
```

Comment:
blocktrans' "verbose format is still supported" and uses `as`, and I guess `{% blocktrans with book|title as book_t as variable %}` would look confusing; bmihelac on the ticket said "to avoid confusion between legacy format for using with."